account_invoice_date/wizard/account_voucher_proforma_date_view.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_account_voucher_proforma_date" model="ir.ui.view">
+        <field name="name">view.account.voucher.proforma.date</field>
+        <field name="model">account.voucher.proforma.date</field>
+        <field name="arch" type="xml">
+            <form>
+                <group>
+                    <field
+                        name="date"
+                        string="Please enter invoice date (should correspond to payment date)"
```

Comment:
```suggestion
                        string="Please enter invoice date"
```
I would avoid the part between brackets, but it is not blocking.

Reply:
Hi, @AaronHForgeFlow I've implemented the changes.